### DISCUSSION TITLE
Найцікавіші новини місяця: жовтень 2024

### DISCUSSION THREAD

**Discussion by malor** [shopokodu/community#74] [Теми для подкасту]
- https://www.phoronix.com/news/Linus-Torvalds-Russian-Devs
  [Linus Torvalds affirms expulsion of russian maintainers • The Register](https://www.theregister.com/2024/10/23/linus_torvalds_affirms_expulsion_of/)
- https://github.com/pypa/gh-action-pypi-publish/discussions/281 (https://peps.python.org/pep-0740/)
- https://www.devjobsscanner.com/blog/top-8-most-demanded-programming-languages/

**Comment by rkiyanchuk** [comment]
[The Disappearance of an Internet Domain](https://every.to/p/the-disappearance-of-an-internet-domain)

[The Static Site Paradox | Loris Cro's Blog](https://kristoff.it/blog/static-site-paradox/)

[The First HTML LSP That Reports Syntax Errors | Loris Cro's Blog](https://kristoff.it/blog/first-html-lsp/)
